test/docker/fedoradev/install-packages.sh

```diff
@@ -16,10 +16,12 @@ dnf -y install /usr/bin/xargs
 while read -r file; do
     case $file in
         /*) printf "%s\n" "$file" ;;
-        *) printf "%s\n" {/usr,}/{,s}bin/"$file" ;;
+        *) printf "%s\n" {/usr,}/bin/"$file" ;;
```

Comment:
The commit message mentions "is being" and the related wiki page does not seem to mention the transition being complete. Is there any drawback from keeping sbin here still? If we remove it, do we lose any coverage?

Reply:
Removed this change for now